charts/mempool/templates/pvc.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "mempool.fullname" . }}
+  annotations:
+    "helm.sh/resource-policy": keep
+  labels:
+    app: "{{ .Chart.Name }}"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
```

Comment:
I'm wondering if we should add these four labels to the other resources as well, or remove them from all. At the moment, it is inconsistent across our charts. For ex we use it in the [fulcrum pvc](https://github.com/galoymoney/charts/blob/ci--pin-gcp-provider-version/charts/fulcrum/templates/pvc.yaml#L9-L12) but not in its [statefulset](https://github.com/galoymoney/charts/blob/ci--pin-gcp-provider-version/charts/fulcrum/templates/statefulset.yaml#L6-L10).

I can see in the concourse chart it's being used uniformly for all resources, and we most likely got it from there.